src/h5web/providers/utils.ts

```diff
@@ -27,14 +26,7 @@ export function createObjectKeyStore<
   });
 
   function wrapMethod<T>(method: (key: string) => T) {
-    return (params: I) => {
-      /* Remove optional params that are explicitely set to `undefined`.
-       * This is to guarantee that, for instance, `{ foo: undefined; }` and `{}`
-       * are both stringified to (and cached with) "{}" */
-      const pickedParams = pickBy(params, (v) => v !== undefined);
-
-      return method(JSON.stringify(pickedParams));
-    };
+    return (params: I) => method(JSON.stringify(params));
```

Comment:
I don't know what I was on at the time, but `JSON.stringify({ foo: undefined })` does return `{}` out of the box... I think it's a left over from a previous attempt I had made with passing params separately instead of together in single `params` object.